### ISSUE TITLE
Display of commons names in alerts 

### ISSUE THREAD

**Issue by turley85** [AtlasOfLivingAustralia/alerts#254]
The common name being displayed is not the ALA preferred vernacular name: 

<img width="641" alt="image" src="https://github.com/user-attachments/assets/80fab60a-a785-4aa1-ac68-bf1738d9e501">

https://bie.ala.org.au/species/https://biodiversity.org.au/afd/taxa/b17002de-8cbb-4bcb-b4a5-048afcfea852#names

Please change so the common name displayed is the ALA preferred vernacular name.

**Comment by adam-collins** [comment]
The preferred name is correct, see the occurrence (please provide a link to the occurrence in the future), and https://namematching-ws.ala.org.au/api/searchByClassification?scientificName=Pseudococcus%20longispinus

See https://github.com/AtlasOfLivingAustralia/bie-index/issues/373 for the related BIE issue.

**Comment by turley85** [comment]
Example of occur ace here: https://biocache.ala.org.au/occurrences/2f2fdd4c-b4de-44da-948a-fdb53023e195

The common name for this is not the ALA preferred vernacular name as per: 
![image](https://github.com/user-attachments/assets/251e3dc6-f671-4ff8-8291-c94cfaf97672) 
https://bie.ala.org.au/species/https://biodiversity.org.au/afd/taxa/b17002de-8cbb-4bcb-b4a5-048afcfea852#

**Comment by adam-collins** [comment]
TLDR
The name matching service is the source of truth, not the BIE species page.

**Comment by turley85** [comment]
Ok, not sure what to do here @adam-collins. The common name in the names matching service doesn't make sense to our users. Who's in-charge of the name-matching service and what names are the preferred name then?

**Comment by adam-collins** [comment]
Seek the taxonomy working group.

**Comment by qifeng-bai** [comment]
@adam-collins Is this ticket solved?

**Comment by qifeng-bai** [comment]
@turley85 @adam-collins 
The Common name uses the vernacularName suppied by Biocache.  For example:

<img width="640" height="199" alt="Image" src="https://github.com/user-attachments/assets/6db1f4db-9fa2-4925-9baf-f1ae37a999d3" /> 

https://biocache-ws.test.ala.org.au/ws/occurrences/cab4a596-f315-4618-9220-60bad2bac3e1
